src/vector_tile_geometry_decoder.ipp

```diff
@@ -248,61 +272,79 @@ void decode_linestring(mapnik::geometry::geometry<typename T::value_type> & geom
         {
             geom = std::move(*itr);
         }
+        else
+        {
+            geom = std::move(mapnik::geometry::geometry_empty());
```

Comment:
I believe it actually is useless, in my mind I always wanted geom set in some manner in the code but then I remembered the default initialization for `mapnik::geometry:geometry<>` is `mapnik::geometry::geometry_empty`